### PR TITLE
chore(compose): add monitoring and kepler metal comparison dashboard

### PR DESCRIPTION
**Pull request by sthaha**
image

# Checklist for PR Author

---

In addition to approval, the author must confirm the following check list:

- [x] Run the following command to format your code:

  ```bash
  make fmt
  ```

- [ ] Create issues for unresolved comments and link them to this PR. Use one of the following labels:
  - `must-fix`: The logic appears incorrect and must be addressed.
  - `minor`: Typos, minor issues, or potential refactoring for better readability.
  - `nit`: Trivial issues like extra spaces, commas, etc.

### CODE REVIEW

**Review by KaiyiLiu1234 — APPROVED** [review]
LGTM

File: manifests/compose/dev/compose.yaml [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+      - ../monitoring/compose.yaml
+      - ./overrides.yaml
+
 services:
-  kepler:
+  kepler-models:
     image: quay.io/sustainable_computing_io/kepler:latest
     ports:
-      - 9100:9100
+      - 19100:9100
```
Comment (vprashar2929) [inline manifests/compose/dev/compose.yaml:11]: @sthaha Any specific reason why are we using `1<port_number>` as host port?

Comment (sthaha) [reply manifests/compose/dev/compose.yaml:11]: Yes, this enables running model-server development compose and kepler dev composes (metal specifically) simultaneously

Comment (vprashar2929) [reply manifests/compose/dev/compose.yaml:11]: But we do run kepler metal as part of model-server compose?

**Pull request merged**